packages/utils/src/cloudwatch/inbound-patient-lambda.ts

```diff
@@ -0,0 +1,213 @@
+/**
+ * Script to query CloudWatch logs for occurrences of "Match: true" over the past two weeks.
+ *
+ * This script performs the following actions:
+ * - Iterates through each day in the past two weeks
+ * - Queries CloudWatch logs for the specified log group and each day's time range
+ * - Counts the number of times "Match: true" appears in the logs for each day
+ * - Outputs the total count per day and the cumulative total
+ */
+
+import { queryLogs } from "./query";
+import { initDbPool } from "@metriport/core/util/sequelize";
+import { QueryTypes } from "sequelize";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import * as fs from "fs";
+import * as path from "path";
+
+/**
+ * Configuration
+ */
+const LOG_GROUP_NAME = "/aws/lambda/IHEInboundPatientDiscoveryV2Lambda";
+const FILTER_PATTERN = '"Match: true"';
+const DAYS_TO_QUERY = 15;
+
+const cxIdMap: Map<string, number> = new Map();
+const recentPatientCountMap: Map<string, number> = new Map();
+const patientIdSet: Set<string> = new Set();
+
+const CX_IDS: string[] = [];
+
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+function writeResultsToCSV(results: any[], filename: string): void {
+  const csvContent = [
+    "CX ID,Total Patients Created,Total Patients Matched,Total Patients Matched (Recent),% of Matches (Recent Patients),% of Recent Patients with Matches",
+    ...results.map(
+      row =>
+        `${row.cxId},${row.totalPatientsCreated},${row.totalPatientsMatched},${row.totalPatientsMatchedRecent},${row.percentageOfMatchesWhichAreRecentPatients},${row.percentageOfRecentPatientsWithMatches}`
+    ),
+  ].join("\n");
+
+  const filePath = path.join(__dirname, filename);
+  fs.writeFileSync(filePath, csvContent);
+  console.log(`Results written to ${filePath}`);
+}
+
+/**
+ * Retrieves patient IDs for specified CX IDs created in the last two weeks.
+ * @param cxIds Array of customer CX IDs.
+ * @returns A map where each key is a CX ID and the value is a set of patient IDs.
+ */
+async function getRecentPatientIds(cxIds: string[]): Promise<Map<string, Set<string>>> {
+  const sqlDBCreds = getEnvVarOrFail("DB_CREDS");
```

Comment:
This is configuration, let's keep it at the top of the file with the other config stuff?